grails-app/services/org/pih/warehouse/data/LocationDataService.groovy

```diff
@@ -39,6 +41,10 @@ class LocationDataService {
                 command.errors.reject("Row ${index + 1}: '${params.name}' already exists in the system. Choose a unique name")
             }
 
+            if (!params.activeOrInactive || (!(params.activeOrInactive instanceof Boolean) && !("true".equalsIgnoreCase(params.activeOrInactive) || "false".equalsIgnoreCase(params.activeOrInactive)))) {
```

Comment:
Call it `active`. How you'd like set `active` field when `active or inactive is true` Is it both active and inactive? :smile:

Reply:
done